JSQMessagesViewController/Views/JSQMessagesTypingIndicatorFooterView.m

```diff
@@ -109,7 +124,123 @@ - (void)configureWithEllipsisColor:(UIColor *)ellipsisColor
     
     [self setNeedsUpdateConstraints];
     
-    self.typingIndicatorImageView.image = [[UIImage jsq_defaultTypingIndicatorImage] jsq_imageMaskedWithColor:ellipsisColor];
+    // Configure correct color and (whether/not) animated
+    [self setTypingIndicatorAnimated:animated :ellipsisColor];
+}
+
+- (void) setTypingIndicatorEllipsisColor:(UIColor*) ellipsisColor
+{
+    self.typingIndicatorView1.backgroundColor = ellipsisColor;
+    self.typingIndicatorView2.backgroundColor = ellipsisColor;
+    self.typingIndicatorView3.backgroundColor = ellipsisColor;
+}
+
+- (void) setTypingIndicatorAnimated:(BOOL) animated :(UIColor*) ellipsisColor
+{
+    // Configure Base Color
+    [self setTypingIndicatorEllipsisColor:ellipsisColor];
+    // Remove existing animator if exist
+    if (self.animator) {
+        [self.animator stopAnimating];
+    }
+    // If animated -> New Animator
+    if (animated) {
+        self.animator = [[JSQMessagesTypingIndicatorAnimator alloc] initWithTypingIndicatorView:self :ellipsisColor];
```

Comment:
yeah. this is weird.

should be `initWithTypingIndicatorView:self color:ellipsisColor`